cartridges/cloudinary_sfra_changes/cartridge/templates/default/product/bundleDetails.isml

```diff
@@ -6,6 +6,19 @@
         assets.addCss('/css/product/detail.css');
     </isscript>
 
+    <isset name="cloudinaryConstants" value="${require('*/cartridge/scripts/util/cloudinaryConstants')}" scope="page" />
+    <isif condition="${pdict.cloudinary && pdict.cloudinary.isEnabled && (pdict.cloudinary.galleryEnabled || pdict.cloudinary.isGalleryEnabled)}">
+        <link rel="preconnect" href="${dw.system.Site.current.preferences.custom.CLDGalleryJSURL}" />
+        <link rel="preload" href="${cloudinaryConstants.CLD_BASE_PATH + '/' + cloudinaryConstants.CLD_VIDEO_RESOURCE_TYPE + '/list/'+ pdict.product.id +'.json'}" as="fetch" type="application/json" crossorigin />
```

Comment:
please remove (everywhere) the `type` attribute - it isn't compatible with `as=fetch`